verify/src/main/java/com/twilio/verify/networking/NetworkProvider.kt

```diff
@@ -42,8 +46,36 @@ class NetworkException constructor(
   )
 }
 
+internal const val CODE_KEY = "code"
+internal const val MESSAGE_KEY = "message"
+internal const val MORE_INFO_KEY = "more_info"
+
 class FailureResponse(
-  val responseCode: Int,
+  val statusCode: Int,
   val errorBody: String?,
   val headers: Map<String, List<String>>?
+) {
+  val apiError: APIError? by lazy {
```

Comment:
because of the exception handler and the map this property does when getting, should it be a method better?

Reply:
Suggestion:
`getApiError()`

Reply:
Using lazy allows us to make the conversion only once. If we replace it with a method, the conversion will be executed every time, wdyt?